src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -407,6 +384,46 @@ protected function instantiateObject(array &$data, $class, array &$context, \Ref
         return new $class();
     }
 
+    /**
+     * @param mixed                $parameterData
+     * @param string               $key
+     * @param \ReflectionParameter $constructorParameter
+     * @param array                $context
+     * @param string|null          $format
+     *
+     * @return mixed value of constructor parameter - an argument
```

Comment:
I'm not sure the docblock lines above provide enough value to be kept, I'd personally suggest to remove them.